client/src/components/DragAndDrop/drag&drop.jsx

```diff
@@ -0,0 +1,81 @@
+import { useState } from "react";
+import { reorderColumn, reorderColumnCardMap } from "./reorder";
+import { DragDropContext, Droppable } from "react-beautiful-dnd";
+import BoardColumn from './column'
+import classes from './assets/item.module.scss'
+import { Box, Grid } from '@material-ui/core';
+import { useBoardContext } from '../../context/useBoardContext';
+
+
+const Board = () => {
+
+    const { boardColumnMap: data, setBoardColumnMap } = useBoardContext();
+    const [columns, setColumns] = useState(data);
+    const [ordered, setOrdered] = useState(Object.keys(data))
```

Comment:
Curious if we have to have a state for tracking the order? Couldn't we just work off the columns list and do the sorting there with array methods?